src/index.html

```diff
@@ -9,6 +9,12 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="watch">
+      <div class="watch-inside">
```

Comment:
```suggestion
      <div class="watch__inside">
```
it can fix this issue
![Screenshot from 2023-08-06 16-38-29](https://github.com/yarkob/layout_stop-watch/assets/33029864/4288131e-7abb-4a30-8d5c-5e9d5e2c4b5e)